docs/source/tutorials/image.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Image Classification with PyTorch and Cleanlab\n"
+    "# Image Classification with PyTorch and Datalab\n"
```

Comment:
```suggestion
    "# Image Classification with PyTorch and Cleanlab\n"
```

Reply:
We want to minimize mentions of Datalab outside of code-level discussions. This is just a variable inside of cleanlab going forward, but we should reference cleanlab whenever we are talking about methods and not specific code